lib/marginalia.rb

```diff
@@ -90,8 +90,8 @@ def exec_update_with_marginalia(sql, name = 'SQL', binds = [])
       exec_update_without_marginalia(annotate_sql(sql), name, binds)
     end
 
-    def execute_and_clear_with_marginalia(sql, *args, &block)
-      execute_and_clear_without_marginalia(annotate_sql(sql), *args, &block)
+    def execute_and_clear_with_marginalia(sql, *args, **kwargs, &block)
+      execute_and_clear_without_marginalia(annotate_sql(sql), *args, **kwargs, &block)
     end
```

Comment:
`execute_and_clear` only has a keyword argument since Rails 5.0:

```suggestion
if ActiveRecord::VERSION::MAJOR >= 5
    def execute_and_clear_with_marginalia(sql, *args, **kwargs, &block)
      execute_and_clear_without_marginalia(annotate_sql(sql), *args, **kwargs, &block)
    end
 else
   def execute_and_clear_with_marginalia(sql, *args, &block)
     execute_and_clear_without_marginalia(annotate_sql(sql), *args, &block)
   end
 end
```